interpreter/binary/decode.ml

```diff
@@ -188,6 +189,7 @@ let ref_type s =
   | -0x17 -> let n = vu32 s in (NonNullable, RttHeapType (var_type s, Some n))
   | -0x18 -> (NonNullable, RttHeapType (var_type s, None))
   | -0x19 -> (Nullable, DataHeapType)
+  | -0x20 -> (Nullable, ArrayHeapType)
```

Comment:
There is a difference with the V8 prototype, as well as MVP.md: there, arrayref, i31ref and dataref are used as shorthands for the non-nullable versions of the types.

Reply:
Yeah, for now I went with what V8 does (required changing some tests), but I opened #266 to perhaps fix this.